packages/admin-ui/client/pages/List/Management.js

```diff
@@ -29,80 +29,70 @@ type Props = {
   onUpdateMany: (*) => void,
   selectedItems: Array<string>,
 };
-type State = {
-  showDeleteModal: boolean,
-  showUpdateModal: boolean,
-};
-
-export default class ListManage extends Component<Props, State> {
-  state = { showDeleteModal: false, showUpdateModal: false };
 
-  closeDeleteModal = () => this.setState({ showDeleteModal: false });
-  openDeleteModal = () => this.setState({ showDeleteModal: true });
-  closeUpdateModal = () => this.setState({ showUpdateModal: false });
-  openUpdateModal = () => this.setState({ showUpdateModal: true });
+export default function ListManage(props: Props) {
+  const { onDeleteMany, onUpdateMany, selectedItems } = props;
+  const [deleteModalIsVisible, toggleDeleteModal] = useState(false);
+  const [updateModalIsVisible, toggleUpdateModal] = useState(false);
```

Comment:
`set*` would probably be better than `toggle*`

Reply:
Sure, no worries